erpnext/stock/doctype/purchase_receipt/purchase_receipt.json

```
@@ -1066,12 +1071,42 @@
    "hidden": 1,
    "label": "Batch No",
    "print_hide": 1
+  },
+  {
+   "fieldname": "license",
+   "fieldtype": "Link",
+   "label": "License",
+   "options": "Compliance Info"
+  },
+  {
+   "depends_on": "doc.docstatus==0;",
+   "fieldname": "reverse_calculate",
+   "fieldtype": "Button",
+   "label": "Reverse Calculate"
+  },
+  {
+   "collapsible": 1,
+   "fieldname": "payment_terms_section",
```

```suggestion
   "fieldname": "payment_terms",
```
Always keep the fieldname same